src/react/ui-elements/FormLayout.jsx

```diff
@@ -32,10 +32,16 @@ export const Title = styled.div`
     display: flex;
 
     text-transform: capitalize;
-    margin-bottom: 40px;
+    margin-bottom: 24px;
     font-size: 19px;
 `;
```

Comment:
Just to be sure: I changed the margin-bottom to 24px for ALL titles (<Title/> Component) is that what I needed to do or is it just in this case?

Reply:
No, just for the case of Replication, which is a specific one.